redfish-core/lib/pcie_slots.hpp

```diff
@@ -229,12 +229,13 @@ inline void linkAssociatedDiskBackplane(
                                 drivePath);
             if (it != assemblyList.end())
             {
+                asyncResp->res.jsonValue["Slots"][index]["Links"]["Oem"]
+                                        ["@odata.type"] = "#OemPCIeSlots.Oem";
                 asyncResp->res.jsonValue["Slots"][index]["Links"]["Oem"]["IBM"]
-                                        ["AssociatedAssembly"] = {
-                    {{"@odata.id",
-                      "/redfish/v1/Chassis/" + chassisId +
-                          "/Assembly#/Assemblies/" +
-                          std::to_string(it - assemblyList.begin())}}};
+                                        ["AssociatedAssembly"]["@odata.id"] =
```

Comment:
any gui changes needed here??

Reply:
Not sure about GUI change.

@Nikhil-Ashoka 
Do you have any  comment on GUI side?

Reply:
Looking at this change, doesn't seem to have a GUI impact, But once this is in, We can check with the GUI.